website/docs/10-overview.md

```diff
@@ -1,15 +1,18 @@
 ---
+slug: /
 id: training-overview
 title: Kubernetes Training Overview
 ---
 
 ## Related Videos
 <VideoContainer
-list={[
-{src: "https://www.youtube-nocookie.com/embed/H5OY7IDjPZU", title: "Overview"},
-]}></VideoContainer>
-
+    list={[{
+      src: "https://www.youtube-nocookie.com/embed/H5OY7IDjPZU", 
+      title: "Overview"
+  }]}
```

Comment:
please align closing brackets with the right indentation. they should match `list`. also you are mixing 4 spaces with 2 spaces indentations. keep using 2 as everywhere else.